### DISCUSSION TITLE
Multi-Tenancy

### DISCUSSION THREAD

**Discussion by suhaib-mousa** [abpframework/abp#15599] [Q&A] [answered]
Hey,
I want to use the Multi-Tenant architecture and I have some concerns:
is the framework supports multi-tenant blob storing where configure multi types of blob storing and the tenant can use the type he/she subscribed to?
What is the substitute for the appsetting for tenants?

**Comment by maliming** [comment, marked as answer]
hi

You can check the documents first.

https://docs.abp.io/en/abp/latest/Multi-Tenancy
https://docs.abp.io/en/abp/latest/Blob-Storing
